README.md

```diff
@@ -29,19 +34,59 @@ refactoring, adding new functionality where it made sense, and generally trying
 with the goal of making it easier to understand and modify. Finally, this release includes a number
 of changes to areas of the book that have given readers difficulties. Enjoy!
 
+Directory Structure
+-------------------
+The organization of this repository is meant to be simple and self-evident at a glance:
 
-Branches
----------
-The `master` branch contains the code at latest release. All ongoing development, with all of the
-latest changes, can be found in the `dev-patch`, `dev-minor`, and `dev-major` branches.
+### books/
+This folder contains the text of the books in the markdeep markup language. This folder also
```

Comment:
Coincidentally, I was thinking today about mixing the actual books (ray*.html) with supporting materials (markdeep.min.js, ackowledgments.md.html). The problem is that it's probably more useful to say that this directory contains the actual books, not just "the text of the books". The latter suggests that these files are used to build the final products, rather than that they are directly consumable as the books themselves.

How about saying something like "this folder contains the three ray tracing books (as HTML), plus supporting materials"?